service/src/main/java/datawave/microservice/audit/auditors/file/config/FileAuditProperties.java

```diff
@@ -9,6 +9,7 @@
 @Validated
 public class FileAuditProperties {
     
+    private String user;
```

Comment:
Does the user need the `@NotEmpty` annotation?

Reply:
Not really. If it's not set the builder will default the value to `datawave` - and the builder doesn't allow `null` to overwrite the current (default) value.